src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -133,7 +133,7 @@ public BlobFeedAction(string expectedFeedUrl, string accountKey, string indexDir
 
             await clientThrottle.WaitAsync();
             string leaseId = string.Empty;
-            AzureBlobLease blobLease = new AzureBlobLease(feed.AccountName, feed.AccountKey, string.Empty, feed.ContainerName, uploadPath, Log, "60", "10");
+            AzureBlobLease blobLease = new AzureBlobLease(feed.AccountName, feed.AccountKey, string.Empty, feed.ContainerName, uploadPath, Log, "15", "3000");
```

Comment:
Can you add a comment in the commit to state what these new times are? Are they all in milliseconds? which one is delay and which one is timeout.

Reply:
Sure, the first one is in seconds and the second in milliseconds.